extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -40,6 +40,9 @@ class AmpLightbox extends AMP.BaseElement {
 
     /**  @private {?function(this:AmpLightbox, Event)}*/
     this.boundCloseOnEscape_ = null;
+
+    /** @private {boolean} */
+    this.isScrollable_ = false;
```

Comment:
can we have a 

this.hasScrollableContent and see if the content is large enough to enable scrolling? this way we would not have to do the gestures unless it is absolutely needed. (basically auto detect).